functions/zoom-webhook.py

```diff
@@ -201,11 +198,24 @@ def construct_sqs_message(payload, context):
     if "on_demand_series_id" in payload:
         sqs_message["on_demand_series_id"] = payload["on_demand_series_id"]
 
+    if zoom_event == "recording.completed":
+        zoom_processing_mins = estimated_processing_mins(
+            sqs_message["start_time"],
+            sqs_message["duration"]
+        )
+        sqs_message["zoom_processing_minutes"] = zoom_processing_mins
```

Comment:
i guess i'm not clear why the value has to get passed along in the sqs message. Is it only because of how it gets logged?

Reply:
Yes, I'm pulling the metric from the sqs message log. This isn't absolutely necessary but the alternative is having a separate log message just for this metric... and I'm trying to not clutter the log messages too much and it doesn't seem to be harmful to have more metadata in the sqs message.